src/styles/index.scss

```diff
@@ -2,2 +1,69 @@
+@import 'variables';
+
+html {
+  font-family: Arial, sans-serif;
+}
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+@each $day, $index in $days {
+  $index: $index + 6;
+  .calendar--start-day-#{$day} > :first-child {
+    margin-left: calc((#{$calendar-day-size} + #{$calendar-gap}) * #{$index});
+  }
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  gap: $calendar-gap;
+  padding: 10px;
+  width: 707px;
+
+  &__day {
+    background-color: $calendar-bg-color;
+    width: $calendar-day-size;
+    height: $calendar-day-size;
+    border: 1px solid #000;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    transition: all 0.5s;
+    text-align: center;
+    box-sizing: border-box;
+
+    @for $day from 1 through 31 {
+      &:nth-child(#{$day})::before {
+        content: '#{$day}';
+        font-family: $calendar-font-family;
+        font-size: $calendar-font-size;
+      }
```

Comment:
The `@for` loop inside the `.calendar__day` class is redundant since it repeats the same loop outside of the class (lines 63-68). You should remove one of these loops to avoid duplication.